src/Vixen.Common/AudioPlayer/FileReader/AudioFileReader.cs

```diff
@@ -20,20 +21,28 @@ public class AudioFileReader : WaveStream, ISampleProvider
         private readonly int _sourceBytesPerSample;
         private readonly long _length;
         private readonly object _lockObject;
+		private static readonly Logger Logging = LogManager.GetCurrentClassLogger();
 
-        /// <summary>
-        /// Initializes a new instance of AudioFileReader
-        /// </summary>
-        /// <param name="fileName">The file to open</param>
-        public AudioFileReader(string fileName)
+		/// <summary>
+		/// Initializes a new instance of AudioFileReader
+		/// </summary>
+		/// <param name="fileName">The file to open</param>
+		public AudioFileReader(string fileName)
         {
-            _lockObject = new object();
-            FileName = fileName;
-            CreateReaderStream(fileName);
-            _sourceBytesPerSample = _readerStream.WaveFormat.BitsPerSample / 8 * _readerStream.WaveFormat.Channels;
-            _sampleChannel = new SampleChannel(_readerStream, false);
-            _destBytesPerSample = 4 * _sampleChannel.WaveFormat.Channels;
-            _length = SourceToDest(_readerStream.Length);
+            try
+            {
+                _lockObject = new object();
+                FileName = fileName;
+                CreateReaderStream(fileName);
+                _sourceBytesPerSample = _readerStream.WaveFormat.BitsPerSample / 8 * _readerStream.WaveFormat.Channels;
+                _sampleChannel = new SampleChannel(_readerStream, false);
+                _destBytesPerSample = 4 * _sampleChannel.WaveFormat.Channels;
+                _length = SourceToDest(_readerStream.Length);
+            }
+            catch (Exception ex)

```

Comment:
There is no real purpose to catch the exception here and then wrap it in another one to re-throw. This should just be allowed to throw since no special handling is occurring.